.github/workflows/dist.yml

```diff
@@ -103,6 +103,36 @@ jobs:
           verbose: true
         if: env.CAN_DEPLOY == 'true'
 
+  noarch_wheels_for_pypi:
+
+    runs-on: ubuntu-latest
+    env:
+      CAN_DEPLOY: ${{ secrets.SAGEMATH_PYPI_API_TOKEN != '' }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install bootstrap prerequisites
+        run: |
+          sudo DEBIAN_FRONTEND=noninteractive apt-get update
+          sudo DEBIAN_FRONTEND=noninteractive apt-get install $(build/bin/sage-get-system-packages debian _bootstrap)
+      - name: make pypi-noarch-wheels
+        run: |
+          ./bootstrap
+          ./configure
+          make pypi-noarch-wheels V=0
+          (mkdir dist && mv upstream/sage*-none-any.whl dist/)
+          ls -l dist
```

Comment:
Is this right?

I get 
```
Built wheels are in venv/var/lib/sage/wheels/
```

Reply:
You are right.

Reply:
I'll also update the actions while at it, hold on please...